README.md

```
@@ -2,35 +2,35 @@
 
 ## Important Links and Resources 📃 -
 
-- [Use Paperpilot](https://paper-pilot.vercel.app/)
+- [Use PaperGo](https://paper-pilot.vercel.app/)
 - [Demo Video]()
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)